### PR TITLE
Update Support for 'facebook' and 'messenger'

### PR DESCRIPTION
**Pull request by mythilytm**
### Checklist
- [ ] Corresponding issue has been opened
- [ ] New tests added
- [ ] Feature flags / configuration added

### Other Related Issues

None

### AFTER YOU MERGE

1. Cut a release tag using the GitHub workflow. Wait for it to complete and the notification to be posted in the #aselo-deploys Slack channel.
2. Comment on the ticket with the release tag version AND any additional instructions required to configure an environment to test the changes.
3. Only then move the ticket into the QA column in JIRA

You are responsible for ensuring the above steps are completed. If you move a ticket into QA without advising what version to test, the QA team will assume the latest tag has the changes. If it does not, the following confusion is on you! :-P

### CODE REVIEW

File: functions/getProfileFlagsForIdentifier.protected.ts [hunk 1]
```diff
@@ -95,8 +95,10 @@ const channelTransformations: { [k: string]: TransformIdentifierFunction[] } = {
   whatsapp: [(s) => s.replace('whatsapp:', ''), phoneNumberStandardization],
   modica: [(s) => s.replace('modica:', ''), phoneNumberStandardization],
   facebook: [(s) => s.replace('messenger:', '')],
+  messenger: [(s) => s.replace('messenger:', '')],
   instagram: [],
   line: [],
+  telegram: [],
```
Comment (mythilytm) [inline functions/getProfileFlagsForIdentifier.protected.ts:101]: telegram was missing

**Review by GPaoloni — COMMENTED** [review]
Can you add more context on why this needed? :thinking:

File: functions/getProfileFlagsForIdentifier.protected.ts [hunk 2]
```diff
@@ -95,8 +95,10 @@ const channelTransformations: { [k: string]: TransformIdentifierFunction[] } = {
   whatsapp: [(s) => s.replace('whatsapp:', ''), phoneNumberStandardization],
   modica: [(s) => s.replace('modica:', ''), phoneNumberStandardization],
   facebook: [(s) => s.replace('messenger:', '')],
+  messenger: [(s) => s.replace('messenger:', '')],
```
[the remark this answers is not in the document]
Comment (mythilytm) [reply functions/getProfileFlagsForIdentifier.protected.ts:98]: [Refer slack for rest of conversation ](https://tech-matters.slack.com/archives/CV4QM23UJ/p1729793503052129)
- We will migrate once programmable chat is deprecated in our codebase

**Review by stephenhand — APPROVED** [review]
LGTM

**Pull request merged**